tasks/93_make-a-person/make-a-person.js

```diff
@@ -20,10 +20,9 @@ export class Person {
     this.fullname = this.first + ' ' + this.last;
   }
   getFirstName() {
-    return this.first = this.fullname.split(' ')[0];
```

Comment:
if you don't update first and second when return this values, and don't update them when set fullName, so there is probably not very good tests which don't cover this case

Reply:
generally it is better to fix :)